tasks/templates/tasks/session_overview.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base/base.html" %}
+
+{% load static %}
+{% load i18n %}
+
+{% block header_title %}
+    {% trans "Overzicht van het takenonderzoek" %} - {{ block.super }}
+{% endblock %}
+
+{% block content %}
+    <div class="uu-inner-container">
+        <div class="col-12">
+            {% with nav_items=study.proposal.available_urls active=3 %}
+                {% include 'base/navigation.html' %}
+            {% endwith %}
+            <h2>{% trans "Overzicht van alle sessies en taken" %}</h2>
+            <p>
+                <em>{% trans "Vanuit dit overzicht is het mogelijk om sessies aan te maken, te bewerken of te verwijderen." %}</em>
+            </p>
+            <div class="session_list">{% include "tasks/session_list.html" %}</div>
```

Comment:
When there aren't any sessions yet, this page is awkwardly empty. I think there should at least be some filler text indicating what to do next. Or a large div that just says "No sessions yet, time to create one!"

This empty space on first visit also kind of invites the SessionStart intro text to be put there. But I'm not 100% on that idea. Perhaps we can ask Desiree for her opinion.

If we put the intro text in an `{% empty %}`, it of course would no longer be visible once a session was created, which is a downside.